docs/website/docs/plus/features/transformations/dbt-transformations.md

```diff
@@ -0,0 +1,225 @@
+    ]
+
+
+# Example orders table which references customer
+@dlt.resource(name="orders", primary_key="id", write_disposition="merge")
+def orders():
+    yield from [
+        {"id": 1, "date": "1-2-2020", "customer_id": 1},
+        {"id": 2, "date": "14-2-2020", "customer_id": 2},
+        {"id": 3, "date": "18-2-2020", "customer_id": 1},
+        {"id": 4, "date": "1-3-2020", "customer_id": 3},
+        {"id": 5, "date": "2-3-2020", "customer_id": 3},
+    ]
+
+# Run your pipeline
+p = dlt.pipeline(pipeline_name="example_shop", destination="duckdb")
+p.run([customers(), companies(), orders(), countries()])
+
+# Define relationships in your schema
+table_reference_adapter(
+    p,
+    "companies",
+    references=[
+        {
+            "referenced_table": "countries",
+            "columns": ["country_id"],
+            "referenced_columns": ["id"],
+        }
+    ],
+)
+
+table_reference_adapter(
+    p,
+    "customers",
+    references=[
+        {
+            "referenced_table": "companies",
+            "columns": ["company_id"],
+            "referenced_columns": ["id"],
+        }
+    ],
+)
+
+table_reference_adapter(
+    p,
+    "orders",
+    references=[
+        {
+            "referenced_table": "customers",
+            "columns": ["customer_id"],
+            "referenced_columns": ["id"],
+        }
+    ],
+)
+
+```
+
+:::note
+Note, that only the relationships that the pipeline is not aware of need to be explicitly passed to the adapter, meaning you don't need to define the parent-child relationships created by `dlt` during the normalization stage, as it will already know about them.
+:::
+
+### 2. Generating your baseline project
+
+Ensure that your dlt pipeline has been run at least once locally or restored from the destination. Then, navigate to the directory where your pipeline is located and, using its name, execute the following command to create a baseline dbt project with dimensional tables for all existing pipeline tables:
+
+```sh
+dlt dbt generate <pipeline-name>
+```
+
+This command generates a new folder named `dbt_<pipeline-name>`, which contains the project with the following structure:
+
+```sh
+dbt_<pipeline-name>/
+├── analysis/
+├── macros/
+├── models/
+│   ├── marts/
+│   │   ├── dim_<pipeline-name>__<table1>.sql
+│   │   ├── dim_<pipeline-name>__<table2>.sql
+│   │   └── dim_<pipeline-name>__<tableN>.sql
+│   ├── staging/
+│   │   ├── sources.yml
+│   │   ├── stg_<pipeline-name>__<table1>.sql
+│   │   ├── stg_<pipeline-name>__<table2>.sql
+│   │   └── stg_<pipeline-name>__<tableN>.sql
+│   ├── <pipeline-name>_dlt_active_load_ids.sql # Used for incremental processing of data
+│   └── <pipeline-name>_dlt_processed_load.sql # Used for incremental processing of data
+├── tests/
+├── dbt_project.yml
+└── requirements.txt
+```
+
+Additionally, in the directory where you ran the generator, you will find a new Python file named `run_<pipeline-name>_dbt.py`, which you can execute to run the project.
```

Comment:
Actually you do not need to generate the dbt project in the same folder as the pipeline, however the credentials need to be in scope. not sure if we need to clarify this.

Reply:
I suggest we leave it as opinionated structure here